packages/open-payments/src/client/index.ts

```diff
@@ -30,11 +30,14 @@ export * from './error'
 export interface BaseDeps {
   axiosInstance: AxiosInstance
   logger: Logger
-  useHttp?: boolean
+  useHttp: boolean
```

Comment:
Made the `useHttp` dependency required so we can get an error whenever we are not passing it.